index.html

```diff
@@ -34,168 +41,178 @@ <h2 class="support-text">Software Developer</h2>
                     coded, don't hestiate to contact me.
```

Comment:
- [ ] GOOD JOB SO FAR 🎉👌🏼👏🏼🙌🏼💥. The header description looks moved a little to the left. To fix this you can remove `margin-right: 10%` from `.text-container` class 👍🏼 .

**Screenshot**
![image](https://user-images.githubusercontent.com/69011963/172994241-00b9a5b9-6d4e-41ea-b1a1-5990f08ac260.png)